### PR TITLE
fix & format example key creation

### PR DESCRIPTION
**Pull request by sh4dowb**
### What was wrong?
was bad

### How was it fixed?
pressing keys on keyboard

### Todo:
fix more

#### Cute Animal Picture

cute doge

### CODE REVIEW

File: docs/web3.eth.account.rst [hunk 1]
```diff
@@ -83,9 +83,9 @@ account you can just generate a random number that acts as a private key.
 
 To create a private key using web3.py and command line you can do:
 
-```shell
-python -c "from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.toHex(acc.privateKey)}, account={acc.address}')"
-```
+.. code-block:: shell
+
+  python3 -c "from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.to_hex(acc._private_key)}, account={acc.address}')"
```
Comment (reedsa) [inline docs/web3.eth.account.rst]: Thanks for this update! Please stick with `python` here to keep it generic and consistent.

Also noticed the return value a few lines down also needs help, perhaps just remove the ticks on that block entirely and add double colon to the line before it `::`.

Comment (sh4dowb) [reply docs/web3.eth.account.rst]: yea formatting is messy, it needs more work for sure, unfortunately I don't know much about `.rst`, but there was also `toHex` and `privateKey` properties used, which doesn't exist on latest version. encountered it while trying it and thought I'd send a quick fix

**Comment by reedsa** [comment]
@sh4dowb got this updated, thanks for your contribution! Will get this merged shortly.

File: docs/web3.eth.account.rst [hunk 2]
```diff
@@ -83,15 +83,13 @@ account you can just generate a random number that acts as a private key.
 
 To create a private key using web3.py and command line you can do:
 
-```shell
-python -c "from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.toHex(acc.privateKey)}, account={acc.address}')"
-```
+.. code-block:: shell
+
+    python -c "from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.to_hex(acc.key)}, account={acc.address}')"
```
Comment (reedsa) [inline docs/web3.eth.account.rst:88]: @pacrob @fselmo before I merge, wanted to double check: is `acc.privateKey` just `acc.key` now?

File: docs/web3.eth.account.rst [hunk 3]
```diff
@@ -83,15 +83,13 @@ account you can just generate a random number that acts as a private key.
...
 
-Which outputs a new private key and an account pair:
+Which outputs a new private key and an account pair::
```
Comment (fselmo) [inline docs/web3.eth.account.rst:90]:
```suggestion
Which outputs a new private key and an account pair:
```

**Review by fselmo — APPROVED** [review]
tiny nit, too many colons but lgtm otherwise 👍🏼

File: docs/web3.eth.account.rst (hunk 2 above)
Comment (sh4dowb) [reply docs/web3.eth.account.rst:88]:
```
>>> from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.to_hex(acc._private_key)}, account={acc.address}')
private key=0x9bb184a212058662a6f8384bc91cxxxxxxxxxxxxxxxxxxxxxxx, account=0xde94644Adb231f69E2aF01f7FF5Ab6182Fd246cc
>>> from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.to_hex(acc.key)}, account={acc.address}')
private key=0xd198207b8c323315c5d3dcc8633xxxxxxxxxxxxxxxxxxxxxxxx, account=0x90bf5A2FCDC10A1168C7DFfC2C3A69a6A3C6BC9F
```

both works I guess, better to use "public" one for sure

Comment (fselmo) [reply docs/web3.eth.account.rst:88]: As of [this commit](https://github.com/fselmo/eth-account/commit/ac9552b2b7f55578c273c4d165bc3567c6989abd), yeah.

File: docs/web3.eth.account.rst (hunk 3 above)
Comment (reedsa) [reply docs/web3.eth.account.rst:90]: That's just to make the format of the following line into a 'literal block'. The backticks were not formatting very well.

Comment (fselmo) [reply docs/web3.eth.account.rst:90]: Ah, I was wondering that actually. TIL 👍🏼

**Pull request merged**